scss/sections/_header.scss

```diff
@@ -1,38 +1,91 @@
 body {
-	margin: 0%;
-	header {
-		width: 100%;
-		background-color: $primary-color;
-		.header_container {
-			min-height: 160px;
-			width: 100%;
-			display: flex;
-			flex-direction: row;
-			align-items: center;
-			margin-left: 5%;
-			.logo {
-				width: 60%;
-			}
-			nav {
-				ul {
-					color: $secondary-color;
-					display: flex;
-					list-style: none;
-					li {
-						padding: 0% 40%;
-						&:hover {
-							cursor: pointer;
-							color: $secondary-color;
-							font-weight: 900;
-						}
-						a {
-							text-decoration: none;
-							color: white;
-							font-size: 1rem;
-						}
-					}
-				}
-			}
-		}
-	}
+  margin: 0%;
+
+  header {
+    width: 100%;
+    background-color: $primary-color;
+
+    .header_container {
+      display: flex;
+      min-height: 160px;
+      width: 90%;
+      margin: 0 5%;
+      flex-direction: row;
+      align-items: center;
+      justify-content: space-between;
+
+      .logo {
+        display: flex;
+      }
+
+      nav {
+        justify-content: end;
+
+        ul {
+          display: flex;
+          color: $secondary-color;
+          list-style: none;
+
+          li {
+            min-width: 100px;
+
+            &:hover {
+              font-weight: 900;
+              color: $secondary-color;
+              cursor: pointer;
+            }
+
+            a {
+              font-size: 23px;
+              text-decoration: none;
+              color: white;
+            }
+          }
+        }
+      }
+    }
+  }
+}
+
+@media (max-width: 767px) {
+  body header .header_container {
```

Comment:
why you need body here?

Reply:
because when i delate body my nav brakes down